ipa.go

```diff
@@ -28,19 +28,18 @@ package verkle
 import (
 	"errors"
 
-	"github.com/crate-crypto/go-ipa/bandersnatch/fr"
 	"github.com/crate-crypto/go-ipa/banderwagon"
 )
 
 type (
-	Fr                        = fr.Element
+	Fr                        = banderwagon.Fr
 	Point                     = banderwagon.Element
 	SerializedPoint           = []byte
 	SerializedPointCompressed = []byte
 )
 
 const (
-	SerializedPointUncompressedSize = 64
+	SerializedPointUncompressedSize = banderwagon.UncompressedSize
 )
 
 func CopyFr(dst, src *Fr) {
```

Comment:
This was initially meant to abstract between KZG and IPA. It can go.

Reply:
> This was initially meant to abstract between KZG and IPA. It can go.

Cool, [done in this commit](https://github.com/gballet/go-verkle/pull/387/commits/55cea862aeb9de4de2ad174c66751e37191de113).